app/uk/gov/hmrc/taxenrolmentassignmentfrontend/testOnly/utils/AccountUtilsTestOnly.scala

```diff
@@ -60,15 +61,17 @@ class AccountUtilsTestOnly @Inject() (
     if (account.identityProviderType == "ONE_LOGIN") {
       for {
         // Insert enrolment-store data
-        _ <- enrolmentStoreServiceTestOnly.insertAccount(account)
-        _ <- account.enrolments.map(enrolment => enrolmentStoreServiceTestOnly.upsertEnrolment(enrolment)).sequence
+        eacdIds <- identityProviderAccountContextConnectorTestOnly.getEacdIds(account.nino.nino)
+        _       <- eacdIds.map(id => oneLoginStubConnectorTestOnly.deleteAccount(id)).sequence
```

Comment:
Yep should have been, moved it but there's issues with it because the postIndividual doesn't have a working test only which we need to get the eacdIds with the nino so I've asked OL team about getting that added or an alternative way to delete them. So will probably be a 2025 job